attestation-service/docs/grpc-as.md

```diff
@@ -2,15 +2,73 @@
 
 `grpc-as` is an Attestation Service application based on gRPC protocol.
 
-## API
+Now the following types of evidence are supported:
+- `azsnpvtpm`: Azure SNP vTPM
+- `sev`: AMD SEV (Not implemented)
+- `sgx`: Intel SGX
+- `snp`: AMD SNP
+- `tdx`: Intel TDX
+- `cca`: Arm CCA
+- `sample`: A fake platform. Only for test and sample
+- `csv`: Hygon CSV
+- `aztdxvtpm`: Azure TDX vTPM
 
-gRPC Attestation Service provides gRPC endpoints which is defined in [protobuf](../protos/attestation.proto).
+## Quick Start
 
-## Usage
+Users can use a [community version of gRPC CoCoAS image](https://github.com/confidential-containers/trustee/pkgs/container/staged-images%2Fcoco-as-grpc) to verify attestation reports.
 
-Here are the steps of building and running gRPC Attestation Service:
+```shell
+# run gRPC CoCoAS server locally
+docker run -d \
+  -v <path-to-attestation-service>/docs/sgx_default_qcnl.conf:/etc/sgx_default_qcnl.conf \ # this qcnl config is used when verifying SGX/TDX quotes
+  -p 50004:50004 \
+  ghcr.io/confidential-containers/staged-images/coco-as-grpc:latest
+```
+
+The `sgx_default_qcnl.conf` is configure the PCCS of Intel platforms. A workable file is given [here](./sgx_default_qcnl.conf) which will connect to Intel's service.
+You can also set the file to connect to another available PCCS. PCCS are usually supported by cloud providers, you can find the steps to configure `/etc/sgx_default_qcnl.conf` for
```

Comment:
One small comment still that I think is worth clarifying here: PCCS is a _caching_ service. If connecting to "Intel's service" (Intel PCS), the requests take place without a caching service proxy. In this case, users are expected to keep a "local" cache on their own (e.g., by storing `.dcap-qcnl` and reusing it).

Reply:
Maybe we can just mention PCS does not keep a cache locally. The reason is that for a common user, either PCS or PCCS do not mean a lot. For advanced users, we have already provided the different ways to set-up/configure PCCS (csp/own/etc.) they can refer to